README.md

```diff
@@ -34,18 +23,19 @@ npm install
  - **[Travis](https://travis-ci.org)** integration and **[Coveralls](https://coveralls.io/)** report
  - (Optional) **Automatic releases and changelog**, using [Semantic release](https://github.com/semantic-release/semantic-release), [Commitizen](https://github.com/commitizen/cz-cli), [Conventional changelog](https://github.com/conventional-changelog/conventional-changelog) and [Husky](https://github.com/typicode/husky) (for the git hooks)
 
-### Importing library
+### Importing libraries
 
-You can import the generated bundle to use the whole library generated by this starter:
+Simply search for `@pixeloven` at [npm](https://www.npmjs.com/) and install the desired packages.
 
+Then you project simply import the install package and away you go!
```

Comment:
Edit: "Then you project..." -> "Then from within your project..."

Also a Nit, its weird that "simply" is used on this line and the one above, I'd pick one.